### PR TITLE
Acceptance extra test

### PR DESCRIPTION
**Pull request by matteofigus**
While reviewing #323 I realised we don't have any fixture for a component that requires dependencies in the acceptance tests suite, and I thought about making a separate PR in order to add that. It should make reviewing #323 easier.

What do you think @matthewdavidson?

### CODE REVIEW

**Comment by matthewdavidson** [comment]
Great shout 👍

File: test/fixtures/components/underscore/.npmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+_package
+package.tar.gz
+node_modules
```
Comment (mattiaerre) [inline test/fixtures/components/underscore/.npmignore]: add extra EOF

File: test/fixtures/components/underscore/server.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports.data = function(context, callback){
+  callback(null, {
+  	magicNumber: _.first([5, 4])
+  });
+};
```
Comment (mattiaerre) [inline test/fixtures/components/underscore/server.js]: add extra EOF

File: test/fixtures/components/underscore/template.jade [hunk 3]
```diff
@@ -0,0 +1 @@
+div The magic number is #{magicNumber}
```
Comment (mattiaerre) [inline test/fixtures/components/underscore/template.jade]: add extra EOF

Comment (matteofigus) [inline test/fixtures/components/underscore/template.jade]: Done

File: test/fixtures/components/underscore/server.js (hunk 2 above)
Comment (matteofigus) [inline test/fixtures/components/underscore/server.js]: Done

File: test/fixtures/components/underscore/.npmignore (hunk 1 above)
Comment (matteofigus) [inline test/fixtures/components/underscore/.npmignore]: Done

**Comment by matteofigus** [comment]
@mattiaerre as discussed I also renamed `underscore` to `underscore-component` to improve readability

**Comment by mattiaerre** [comment]
LGTM 👍

**Pull request merged**